### DISCUSSION TITLE
How to open closed indices again?

### DISCUSSION THREAD

**Discussion by aa18afr** [Security-Onion-Solutions/securityonion#11252] [General] [answered]
Hi All,

I would like to know if indice is closed all the data from SOC dashboard gets lost of that day indice. Now, if closed indice needs to be reopened then how it can be done? Please provide support, many thanks.

**Comment by dougburks** [comment, marked as answer]
Wes responded to your other discussion yesterday:
https://github.com/Security-Onion-Solutions/securityonion/discussions/11247

In the future, please avoid creating duplicate discussions.
